src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_help.py

```diff
@@ -0,0 +1,379 @@
+    examples:
+        - name: Shows the Eventhub details.
+          text: az eventhubs eventhub show --resource-group myresourcegroup --namespace-name mynamespace --name myeventhub
+"""
+
+helps['eventhubs eventhub list'] = """
+    type: command
+    short-summary: List the EventHub by Namepsace
+    examples:
+        - name: Get the Eventhubs by Namespace.
+          text: az eventhubs eventhub list --resource-group myresourcegroup --namespace-name mynamespace
+"""
+
+helps['eventhubs eventhub delete'] = """
+    type: command
+    short-summary: Deletes the Eventhub
+    examples:
+        - name: Deletes the Eventhub
+          text: az eventhubs eventhub delete --resource-group myresourcegroup --namespace-name mynamespace --name myeventhub
+"""
+
+helps['eventhubs eventhub authorization-rule create'] = """
+    type: command
+    short-summary: Creates Authorization Rule for the given Eventhub
+    examples:
+        - name: Creates Authorization rules
+          text: az eventhubs eventhub authorization-rule create --resource-group myresourcegroup --namespace-name mynamespace --event-hub-name myeventhub --name myauthorule --rights Listen
+"""
+
+helps['eventhubs eventhub authorization-rule update'] = """
+    type: command
+    short-summary: Updates Authorization Rule for the given Eventhub
+    examples:
+        - name: Updates Authorization rules
+          text: az eventhubs eventhub authorization-rule update --resource-group myresourcegroup --namespace-name mynamespace --event-hub-name myeventhub --name myauthorule --rights Send
+"""
+
+helps['eventhubs eventhub authorization-rule show'] = """
+    type: command
+    short-summary: shows the details of Authorization Rule
+    examples:
+        - name: shows the details of Authorization Rule
+          text: az eventhubs eventhub authorization-rule show --resource-group myresourcegroup --namespace-name mynamespace --event-hub-name myeventhub --name myauthorule
+"""
+
+helps['eventhubs eventhub authorization-rule list'] = """
+    type: command
+    short-summary: shows the list of Authorization Rule by Eventhub
+    examples:
+        - name: shows the list of Authorization Rule by Eventhub
+          text: az eventhubs eventhub authorization-rule show --resource-group myresourcegroup --namespace-name mynamespace --event-hub-name myeventhub
+"""
+
+helps['eventhubs eventhub authorization-rule  keys list'] = """
+    type: command
+    short-summary: Shows the connection strings of Authorization Rule for the Eventhub.
+    examples:
+        - name: Shows the connection strings of Authorization Rule for the eventhub.
+          text: az eventhubs eventhub authorization-rule list-keys --resource-group myresourcegroup --namespace-name mynamespace --event-hub-name myeventhub --name myauthorule
+"""
+
+helps['eventhubs eventhub authorization-rule  keys renew'] = """
+    type: command
+    short-summary: Regenerate the connection strings of Authorization-Rule for the namespace.
```

Comment:
Should `Authorization-Rule` be hyphenated or not.
Here, it's hyphenated but below it is not.

Reply:
talked with PM, it should be without space in description. changed to Authorizationrule.